src/client/components/Footer/Footer.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import logo from '*../../../src/client/assets/images/logo.png';
+import send from '*../../../src/client/assets/images/emailsend.png';
+import './Footer.css';
+
+function Footer() {
+  return (
+    <div>
+      <footer>
+        <ul>
+          <li className="footer-list">
+            <div className="logo">
+              <img src={logo} alt="logo" />
+            </div>
+          </li>
+          <li className="footer-list">
+            <div className="footer-text">
+              <h4>COMPANY</h4>
+              <a href="/About">About us</a>
+              <br />
+              <a href="/Contact">Contact us</a>
+            </div>
+          </li>
+          <li className="footer-list">
+            <div className="footer-text">
+              <a href="/sign-up">
```

Comment:
I think it is better to use <Link to=""/> react router dom instead

Reply:
Can i use like 
<Link to="/sign-up"><h4>CREATE ACCOUNT</h4></Link>